resources/whales/tasks/task4.ipynb

```diff
@@ -32,52 +32,76 @@
    "metadata": {},
    "source": [
     "# Your Solution\n",
-    "Your solution function should be called solution. In this case we leave it for consistency but you don't need to do anything with it. \n",
+    "Your solution function should be called solution.\n",
     "\n",
-    "CONFIG is a dictionary with all parameters that you want to pass to your solution function."
+    "CONFIG is a dictionary with all parameters that you want to pass to your solution function. In this case we leave it for consistency but you don't need to do anything with it. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "CONFIG={'regularize':True,\n",
     "        'weight_decay_conv2d': 0.0005,\n",
     "        'weight_decay_linear': 0.01},\n",
     "\n",
-    "def solution(model, regularize, weight_decay_conv2d, weight_decay_linear, ):\n",
-    "    \"\"\"\n",
-    "    Implement the weight here.\n",
+    "def solution(model, regularize, weight_decay_conv2d, weight_decay_linear):\n",
+    "    \"\"\"Solution for task 4.\n",
+    "    \n",
     "    This function should return an object that can be passed to the pytorch optimizer as in:\n",
     "    \n",
     "    optimizer = optim.SGD(model.parameters(), lr=0.1, weight_decay=0.001)\n",
     "    \n",
     "    In this case model.parameters() is our parameter list but it puts the same regularization\n",
     "    on all the neurons in all the layers. Adjust it to our case.\n",
+    "    \n",
+    "    Parameters\n",
```

Comment:
Hey @pknut @jakubczakon on Monday we have to decide which style we take: [Google-style](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) or [NumPy-style](http://www.sphinx-doc.org/en/master/ext/example_numpy.html).

Reply:
@pknut @jakubczakon @rafajak we have decided that we are going to use Google style in all our projects :)